locksmith/__tests__/controllers/blockController.test.ts

```diff
@@ -1,12 +1,12 @@
+import nock = require('nock')
```

Comment:
In the future it may be worth settling on ES6 module imports for everything, but that's not a blocker for this update.